src/session/src/context.rs

```diff
@@ -43,6 +43,10 @@ const CURSOR_COUNT_WARNING_LIMIT: usize = 10;
 #[builder(build_fn(skip))]
 pub struct QueryContext {
     current_catalog: String,
+    /// mapping of RegionId to SequenceNumber, for snapshot read, meaning that the read should only
+    /// container data that was committed before(and include) the given sequence number
+    /// this field will only be filled if extensions contains a pair of "snapshot_read" and "true"
+    snapshot_seqs: Arc<RwLock<Option<HashMap<u64, u64>>>>,
```

Comment:
Do we have a suitable type alias to replace these two u64s?

Reply:
> Do we have a suitable type alias to replace these two u64s?

both `RegionId` and `SequenceNumber` is defined in downstream crate(`store-api`), so no alias for them in `session` crate